packages/stryker-javascript-mutator/src/JavaScriptMutator.ts

```diff
@@ -41,6 +43,16 @@ export default class JavaScriptMutator implements Mutator {
     return mutants;
   }
 
+  private hasValidExtension(file: TextFile): boolean {
+    for (let extension of this.knownExtensions) {
```

Comment:
We could also use `return this.knownExtensions.some(extension => path.extname(file.name) === extension);`